Go_Refined_Code/static/style.css

```
@@ -18,6 +18,8 @@
   --accent-ring: rgba(200, 169, 110, 0.22);
   --error:       #C26060;
   --error-bg:    rgba(194, 96, 96, 0.09);
+  --success:     #4f9e6f;
+  --success-bg:  rgba(79, 158, 111, 0.09);
   --success:     #4FA87A;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "--success" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)